plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/CustomServerEvaluationStrategy.java

```diff
@@ -302,7 +382,7 @@ protected void initPortMapping() {
         /**
          * Gets default external address.
          */
-        protected String getExternalAddress() {
+        public String getExternalAddress() {
```

Comment:
yes, because I pulled up the method in the `RenderingEvaluation` interface, since it is now also useful in `CustomServerEvaluationStrategy.getExternalAddressesAndPorts()` (in case `cheDockerCustomExternalTemplate` is `null`)